radiology.csl

```diff
@@ -180,6 +180,11 @@
     <choose>
       <if type="article-journal article-magazine" match="any">
         <text variable="volume"/>
+        <choose>
+          <if match="any" variable="issue">
+            <text variable="issue" prefix="(" suffix=")"/>
+          </if>
+        </choose>
```

Comment:
The CSL specification is a bit dense, but it's there:

http://docs.citationstyles.org/en/stable/specification.html#affixes
> The attribute value is either added before (prefix) or after (suffix) the output of the element carrying the attribute, but *affixes are only rendered if the element produces output*.